sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 3.2.0b1 (Unreleased)
+## 3.2.0b1 (2021-10-05)
```

Comment:
was hoping to add a few sentences in here about the changes with this version. Want to just copy the snippet from the readme?

Reply:
Done!